modAionImpl/src/org/aion/zero/impl/blockchain/StandaloneBlockchain.java

```diff
@@ -609,6 +613,13 @@ public synchronized void setBlockNumber(long blockNumber) {
                                 .withEnergyLimit(this.genesis.getNrgLimit())
                                 .withDifficulty(this.genesis.getDifficulty())
                                 .withTimestamp(0)
+                                // With default header format.
+                                .withSolution(EMPTY_SOLUTION)
```

Comment:
I would prefer adding a method called `.withEmptySolution()` to the builder, instead of importing `EMPTY_SOLUTION`. Similarly for all the methods below.